api.py

```diff
@@ -8,4 +8,4 @@
 application.register_blueprint(status_blueprint)
 
 if __name__ == '__main__':
-    application.run()
+    application.run(port=5001)
```

Comment:
Wont this break the docker container?

Reply:
Docker uses `flask run`

Reply:
Shouldn't this be done via a setting?  Seems odd to me to hard code it here

Reply:
This line exists purely for running locally, not sure if needs to be a setting if it doesn't change per environment.